flink-runtime/src/main/java/org/apache/flink/runtime/highavailability/HighAvailabilityServicesUtils.java

```diff
@@ -82,6 +80,31 @@ public static HighAvailabilityServices createAvailableOrEmbeddedServices(
         }
     }
 
+    @Nonnull
```

Comment:
Isn't the `@Nonnull` obsolete here again?

Reply:
Yes. Will remove it.